src/main/java/com/ericsson/ei/services/SubscriptionService.java

```diff
@@ -57,7 +58,7 @@
     private String repeatFlagHandlerCollection;
 
     @Value("${ldap.enabled}")
-    private boolean authenticate;
+    private boolean ldapEnabled;
```

Comment:
if you change name here, then you can change the name in all other places where the "authenticate" name is used. So we use same name in all places.

Reply:
Done